packages/argon2-cffi-bindings/meta.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: argon2-cffi-bindings
+  version: 21.2.0
+  top-level:
+    - _argon2_cffi_bindings
+source:
+  url: https://files.pythonhosted.org/packages/b9/e9/184b8ccce6683b0aa2fbb7ba5683ea4b9c5763f1356347f1312c32e3c66e/argon2-cffi-bindings-21.2.0.tar.gz
+  sha256: bb89ceffa6c791807d1305ceb77dbfacc5aa499891d2c55661c6459651fc39e3
+about:
+  home: https://github.com/hynek/argon2-cffi-bindings
+  PyPI: https://pypi.org/project/argon2-cffi-bindings
+  summary: Low-level CFFI bindings for Argon2
+  license: MIT
+extra:
+  recipe-maintainers:
+    - alarmfox
```

Comment:
Nitpicking, could you please move these fields to the end of the recipe (and similarly for other recipes)? We usually format the recipe with the order of "package metadata -> source -> requirements -> build -> about -> extra".

Reply:
Done in the following commits